### PR TITLE
[UI] Add initial UI to show weather data 

### LINKED ISSUE #6: [UI] Add initial UI to show weather data
## Why

Need to add initial UI to show weather data.
 
## Acceptance Criteria?

- Create UI to show temperature, humidity and an input field to enter city name.
- Make sure the app shows new UI after running.

### PR DESCRIPTION
**Pull request by Shayokh144**
https://github.com/nimblehq/ci-cd-sample-ios/issues/6

## What happened

Add UI for taking city name input and showing temperature and humidity label.
 
## Insight

- Add launcher image
- Add textfield to take input
- Add 2 labels to show outputs
`CI is not triggered for this PR to save credit for Deployment task.`
 
## Proof Of Work
simulator_screenshot_81F4AF16-12FA-4958-844C-98F539DD1471

### CODE REVIEW

File: WeatherToday/Sources/Presentation/Modules/WeatherViewController.swift [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    init () {
+        cityTextField = UITextField()
+        temperatureLabel = UILabel()
+        humidityLabel = UILabel()
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
```
Comment (blyscuit) [inline WeatherToday/Sources/Presentation/Modules/WeatherViewController.swift]: I think you can use lazy var. This way you don't need the init.
```
lazy private var cityTextField = UITextField()
```

**Review by blyscuit — APPROVED** [review]

File: WeatherToday/Sources/Presentation/Modules/WeatherViewController.swift (hunk 1 above)
Comment (Shayokh144) [reply WeatherToday/Sources/Presentation/Modules/WeatherViewController.swift]: fixed in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/17/commits/4dc7eeab72cc22c49dae1be2fe5e4e1b82d5d306)

File: WeatherToday/Sources/Presentation/Modules/WeatherViewController.swift [hunk 2]
```diff
@@ -0,0 +1,94 @@
+import UIKit
+
+final class WeatherViewController: UIViewController {
+
+    lazy private var cityTextField: UITextField = UITextField()
```
Comment (vnntsu) [inline WeatherToday/Sources/Presentation/Modules/WeatherViewController.swift]: Although we haven't had a convention for the order of access control yet, we can refer the raywenderlich's access control [convention](https://github.com/raywenderlich/swift-style-guide#access-control).
So following:
> Use access control as the leading property specifier. The only things that should come before access control are the static specifier or attributes such as @IBAction, @IBOutlet and @discardableResult.

`private` should be placed ahead of `lazy`.

File: WeatherToday/Sources/Presentation/Modules/WeatherViewController.swift [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    }
+
+    private func updateConstraintForCurrentOrientation() {
+        if UIDevice.current.orientation.isLandscape {
+            portraitTopConstraint.isActive = false
+            landscapeTopConstraint.isActive = true
+        } else {
+            landscapeTopConstraint.isActive = false
+            portraitTopConstraint.isActive = true
+        }
```
Comment (vnntsu) [inline WeatherToday/Sources/Presentation/Modules/WeatherViewController.swift]:
```suggestion
        let isLandscape = UIDevice.current.orientation.isLandscape
        portraitTopConstraint.isActive = !isLandscape
        landscapeTopConstraint.isActive = isLandscape
```

**Review by vnntsu — COMMENTED** [review]
@Shayokh144  I've just noticed that we're using branch name `feature-add-initial-ui-to-show-weather-data` or `chore-xxx`, it's pretty bit wrong if compared to our process.

I prefer to follow pattern: `feature/#{issue_id}` or `chore/#{issue_id}`. What do you think?

File: WeatherToday/Sources/Presentation/Modules/WeatherViewController.swift (hunk 2 above)
Comment (Shayokh144) [reply WeatherToday/Sources/Presentation/Modules/WeatherViewController.swift]: resolved in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/17/commits/f6bd9482b6d7960d05e03b5d079bf5e2a28fe439)

File: WeatherToday/Sources/Presentation/Modules/WeatherViewController.swift (hunk 3 above)
Comment (Shayokh144) [reply WeatherToday/Sources/Presentation/Modules/WeatherViewController.swift]: resolved in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/17/commits/f6bd9482b6d7960d05e03b5d079bf5e2a28fe439)

File: WeatherToday/Sources/Presentation/Modules/WeatherViewController.swift (hunk 2 above)
Comment (Shayokh144) [reply WeatherToday/Sources/Presentation/Modules/WeatherViewController.swift]: @vnntsu about the naming of branch, I think you are right. `feature/#{issue_id}` this pattern will be more readable, I will follow this for future branches.

**Review by vnntsu — APPROVED** [review]
LGTM!

**Pull request merged**